stacks/rust/run/Dockerfile

```diff
@@ -0,0 +1,8 @@
+ARG version=tip
```

Comment:
See my note above about using just the `ubi8-minimal-{{VERSION}}` image as the run image for Rust.